src/App.tsx

```diff
@@ -333,8 +336,9 @@ function App() {
                 </p>
                 {hiddenChannelOptions.length > 0 && (
                   <p className="italic text-xs mb-4 w-full text-center">
-                    Deposit more sats to unlock {hiddenChannelOptions.length}{" "}
-                    more nodes
+                    {hiddenChannelOptions.length} nodes are hidden because they
+                    a higher minimum channel size than your current onchain
+                    balance.
```

Comment:
why is that useful information? what am I supposed to do with that info?

Reply:
because if the user only deposits 20,000 sats they would think Alby only has one recommended node. Is that ok?

Reply:
The user should know that they should deposit more sats to have more options.

Reply:
What about "Increase your onchain balance to see more recommended nodes". CC @MoritzKa